pkgs/development/libraries/libtoxcore/default.nix

```diff
@@ -18,8 +18,10 @@ stdenv.mkDerivation rec {
     "-DBOOTSTRAP_DAEMON=ON"
   ];
 
-  buildInputs = [
-    libsodium libmsgpack ncurses
+  buildInputs = [ ncurses ];
+
+  propagatedBuildInputs = [
```

Comment:
I believe propagatedBuildInputs only apply to python packages.

Reply:
They're used way more in python packaging, while they're normally reserved for ensuring that adding `a` to inputs also brings in `b`, as in this case.

Reply:
`ratox` should not explicitly link with `libsodium` and `libmsgpack` because it does not include their headers and is not linked with them statically. I have removed the change to `libtoxcore`.